src/i18n/definitions.js

```diff
@@ -1023,6 +1026,7 @@ let strings = new LocalizedStrings(
       classroomTab: "Klasse",
       bookmarkedTab: "Bogmærker",
       saved: "Gemt",
+      searches: "Mine Søgninger",
```

Comment:
We are not using the Localization strings anymore, so you don't have to add the localized version of the strings.